src/reducers/auth.ts

```diff
@@ -0,0 +1,25 @@
+import * as Constants from 'constants/auth'
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+type ActionType = {
+  type: string
+  payload: any
+}
+
+const AuthReducer = (state: Record<string, unknown>, action: ActionType): Record<string, unknown> => {
+  switch (action.type) {
+    case Constants.AUTH: {
+      const { access_token, refresh_token, token_type } = action.payload.attributes
+      localStorage.setItem('access_token', access_token)
+      localStorage.setItem('refresh_token', refresh_token)
+      localStorage.setItem('token_type', token_type)
+
+      return { ...state }
```

Comment:
The `state` is no defined anywhere and this return is `empty`! 🤔 Why are we returning an empty state?

Reply:
After the fix e187091b02cc6415f847925c96333565b9580b51. When we define the return type to the function, we need to return something accord to the return type which is `AuthState` in this case.

Reply:
@gutakk Got it. But don't we need to turn the `isAuthenticated` to true in the `AuthState` as login is successful here.

Reply:
Yes, we need to. I updated it in https://github.com/gutakk/react-surveyor/pull/40